gokart/build.py

```diff
@@ -0,0 +1,58 @@
+import logging
+import os
+import sys
+from logging import getLogger
+from typing import Any, Optional
+
+import luigi
+
+from gokart.run import _check_config, _read_environ
+from gokart.task import TaskOnKart
+
+
+class HideLogger:
+    def __init__(self, verbose: bool):
+        self.verbose = verbose
+        self.logger = getLogger(__name__)
+        self.default_level = self.logger.level
+
+    def __enter__(self):
+        if not self.verbose:
+            logging.disable(sys.maxsize)
+            self.logger.setLevel(logging.CRITICAL)
+        return self
+
+    def __exit__(self, exception_type, exception_value, traceback):
+        logging.disable(self.default_level)
+        self.logger.setLevel(self.default_level)
+
+
+def _get_output(task: TaskOnKart) -> Any:
+    output = task.output()
+    if type(output) == list:
+        return [x.load() for x in output]
+    return output.load()
+
+
+def add_config(file_path: str):
```

Comment:
@vaaaaanquish 
[ASK]
Is this method should be in `build` package?
It seems this is just a helper not necessarily required for gokart.build.

Reply:
Yes, I created `utils.py` and moved some things around.